shoppingfeed.php

```diff
@@ -55,4 +55,4 @@ function init() {
 	\WP_CLI::add_command( 'shopping-feed feed-generation', '\\ShoppingFeed\ShoppingFeedWC\Cli\FeedGeneration' );
 }
 
-\add_action( 'plugins_loaded', __NAMESPACE__ . '\\init', 100 );
+\add_action( 'init', __NAMESPACE__ . '\\init', 100 );
```

Comment:
Hello, why have changed this hook ?

Reply:
Hello,

Sorry for the response delay. in fact, activating the plugin triggered an error for us. By modifying the delay in which the callback function launches, this solved the problem.

Reply:
What was the error message ? Because this is the right hook :)